samples-generator/lib/createBuildingsTile.js

```diff
@@ -189,3 +226,46 @@ function generateBatchTableBinary(buildings) {
         binary : batchTableBinary
     };
 }
+
+function generateBatchTableBinary(buildings) {
+    var buildingsLength = buildings.length;
+    var cartographicBuffer = Buffer.alloc(buildingsLength * 3 * sizeOfDouble);
+    var codeBuffer = Buffer.alloc(buildingsLength * sizeOfUint8);
+
+    var batchTableJson = {
+        cartographic : {
+            name : 'CartographicBuffer',
+            byteOffset : 0,
+            byteLength : cartographicBuffer.length,
+            componentType : 0x1406, // TODO: Logical error, not sure what to do, 'DOUBLE' was here originally but GLTF only supports Floats
```

Comment:
Let's remove the TODO for now and go with 32-bit floats until we have a better solution. Make sure that `cartographicBuffer` uses `sizeOfFloat` instead of `sizeOfDouble` above and uses `writeFloatLE` instead of `writeDoubleLE` below.